README.md

```diff
@@ -1,87 +1,69 @@
 # querido-diario-api
 
-Welcome to the Querido Diário API project! The goal of this repository is keep
-the source code used to build the API used to make available the gazettes crawled
-by the [Querido Diário](https://github.com/okfn-brasil/querido-diario) project. 
+______________________________________
 
-## Requisites
+_[Click here](languages/en-US/README.md) to read this article in english._
+______________________________________
 
-The whole project is build and run inside containers and is supported only in 
-Linux environments (for now). As we run everything inside containers you just need
-to install [`podman`](https://podman.io/) to run locally in your machine. If 
-you are not familiarize with `podman`, you can think it as a lightweight 
-`docker`/`docker-compose`. It exists in almost all Linux distributions packages 
-repositories. 
+Seja bem-vinda(o) ao projeto Querido Diário API! O objetivo deste repositório é manter o 
+código-fonte da API que disponibiliza os diários oficiais raspados pelo [Querido Diário](https://github.com/okfn-brasil/querido-diario). 
 
-Please, check podman [documentation](https://podman.io/getting-started/installation.html) 
-to see how to installed it in your environments. If you face some difficulties 
-with podman, let us know!
+## Requisitos
 
-## Build
+Todo o projeto é construído e executado em contêineres e é compatível apenas com ambientes Linux 
+(por enquanto). Como executamos tudo dentro de contêineres, você só precisa instalar o [`podman`]
+ (https://podman.io/) para rodar localmente em sua máquina. Se você não está familiarizado com o 
+ `podman`, você pode pensá-lo como um `docker` /` docker-compose` mais “leve”. Ele existe em 
+ quase todos os repositórios de pacotes de distribuição Linux.     
+Por favor, consulte a  [documentação do podman](https://podman.io/getting-started/installation
```

Comment:
Pra fazer um novo parágrafo a gente precisa colocar uma linha em branco entre as duas linhas.

---
### English

To make a new paragraph we have to put an empty line between the two lines.